README.md

```diff
@@ -1 +1,86 @@
-# tv-maze
\ No newline at end of file
+# Tv maze
+
+Your site for watching any movie for free.
+
+## Built With
+
+- Major languages: HTML, CSS & Javascript
+- Technologies used: Linters, Git and Github, webpack and jest
+
+## Live Demo
+
+[https://john-kibirige.github.io/tv-maze/](https://john-kibirige.github.io/tv-maze/)
+
+## Video description
+
+[Video description](https://drive.google.com/file/d/1I-L7Mm9XUsEtWLLenc9ZfN78tgYzMHJ7/view?usp=sharing)
+
```

Comment:
You guys did well on this project but there are a few details you skipped in the video presentation. Kindly take note of the points listed below about the video presentation:

- [ ] The student recorded a video presenting the features of the project. ( Please talk about the key features of the project )
- [ ] The maximum length of the video is 5 minutes. (Your video lasted for 5.11mins which is above the max time for the video. Kindly do your video within 5 minutes or 5 max 👍)
- [ ] The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear. ( I will suggest you consider this point as the first point. You introduce yourselves, kind of like, "This is our JavaScript capstone project that required us to build an app that interacts with an external API and we used the tv-maze  API. We worked on these features: list the features of the project 👍")
- [ ] All the members in the video should appear in the video and talk about the project.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making. ( At least one member should open a section of their code that he is proud of implementing. Don't explain everything about the code. You can show the code and briefly talk about what that code does in the app 👍)

**Kindly redo the video and follow the guides above in presenting your project**